SA3/qa_web/tests/test_views.py

```diff
@@ -68,6 +82,15 @@ def test_edit_profile(self):
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, "Enter a valid date.")
 
+    def test_displaying_profile(self):
+        self._login()
+        user = User.objects.get_by_natural_key(credentials['username'])
+        num_answers, comments_per_answer = 5, 2
+        q = _populate_db(user, num_answers, comments_per_answer)
+        self.assertIsNotNone(q)
+        response = self.client.get('/profile/{}/'.format(user.id))
+        self.assertEqual(response.status_code, 200)
```

Comment:
Might want to add more extensive assertion such as number of recent posts displayed on the user profile? Here it should be `1 + 5*2` I believe.

Reply:
It seems the User profile does not shows comments, so I can just assert the number of questions and answers.